stratum/hal/bin/bcm/standalone/BUILD

```diff
@@ -196,3 +203,99 @@ pkg_deb(
     package = "stratum-bcm-sdklt",
     version = "0.0.1",
 )
+
+container_image(
+    name = "stratum_bcm_opennsa_docker_with_deb",
+    base = "@minideb//image",
```

Comment:
is the Debian version defined somewhere?

Reply:
In WORKSPACE:

```bazel
container_pull(
  name = "minideb",
  registry = "index.docker.io",
  repository = "bitnami/minideb",
  tag = "stretch",
)
```